app/src/frontend/pages/leaderboard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+
+import './leaderboard.css';
+
+    interface Leader {
+        number_edits: string;
+        username: string;
+    }
+
+
+    interface LeaderboardProps {
+    }
+
+
+    interface LeaderboardState {
+        leaders: Leader[];
+        fetching: boolean;
+
+        //We need to track the state of the radio buttons to ensure their current state is shown correctly when the view is (re)rendered
+        number_limit: number;
+        time_limit: number;
+    }
+
+
+class LeaderBoardPage extends Component<LeaderboardProps, LeaderboardState> {
```

Comment:
let's make the class and interface names consistent in terms of `Leaderboard`/`LeaderBoard` casing (not sure which one is more correct)

Reply:
Fixed